web/src/pages.rs

```diff
@@ -162,11 +177,11 @@ pub async fn my_categories(
 
     let categories = sqlx::query!(
         "SELECT DISTINCT c.* 
-         FROM category c
-         JOIN categorymarkdown cm ON c.category_id = cm.category_id
-         JOIN markdown m ON cm.markdown_id = m.markdown_id
-         JOIN pagesnapshot ps ON m.page_snapshot_id = ps.page_snapshot_id
-         JOIN pages p ON ps.page_id = p.page_id
+         FROM categories c
+         JOIN markdown_categories cm USING (category_id)
+         JOIN markdowns m USING (markdown_id)
+         JOIN page_snapshots ps USING (page_snapshot_id)
+         JOIN pages p USING (page_id)
```

Comment:
I think this `USING` syntax is nice and we specifically named the columns so that we could use it!